src/lib/components/particles/stacked-bar/index.jsx

```diff
@@ -1,12 +1,14 @@
-import { useRef } from 'preact/hooks'
+import { useRef, useMemo } from 'preact/hooks'
 import defaultStyles from './style.module.css'
 import { mergeStyles } from '$styles/helpers/mergeStyles'
+import { positionLabels } from './stackedBarUtil'
 
 export function StackedBar({
   stack,
   width,
   height,
-  hideLabels = true,
+  hideLabels = false,
+  labelType = 'hanging',
```

Comment:
Can we make a const with the possible values for label type? Otherwise it's difficult to know which values are valid.